namespaces/collectd.yml

```diff
@@ -51,29 +66,54 @@ namespace:
           TODO
 
     - name: ps_vm
-      type: integer
+      type: group
```

Comment:
@sradco, let me see if I can clarify.  Here is sample input JSON from `collectd`:

```
{
  "values": [ 1019961344 ],
  "dstypes": [ "gauge" ],
  "dsnames": [ "value" ],
  "interval": 10.0,
  "host": "dhcp-0-135.tlv.redhat.com",
  "plugin": "processes",
  "plugin_instance": "collectd",
  "type": "ps_vm",
  "type_instance": ""
}
```

Then would the normalized form suggested by this commit be:

```
{
  "hostname": "dhcp-0-135.tlv.redhat.com",
  "collectd": {
    "ps_vm": {
      "value": 1019961344
    },
    "dstypes": [ "gauge" ],
    "interval": 10.0,
    "plugin": "processes",
    "plugin_instance": "collectd"
  }
}
```

If so, adding a field name `value` seems unnecessary.  Could you consider this?

```
{
  "hostname": "dhcp-0-135.tlv.redhat.com",
  "collectd": {
    "ps_vm": 1019961344,
    "dstype": "gauge",
    "interval": 10.0,
    "plugin": "processes",
    "plugin_instance": "collectd"
  }
}
```